sdk/helper/testcluster/docker/environment_test.go

```diff
@@ -0,0 +1,35 @@
+package docker
+
+import (
+	"testing"
+)
+
+func TestSettingEnvsToContainer(t *testing.T) {
+	expectedEnv := "TEST_ENV=value1"
+	expectedEnv2 := "TEST_ENV=value2"
+	opts := &DockerClusterOptions{
+		ImageRepo: "hashicorp/vault",
+		ImageTag:  "latest",
+		Envs:      []string{expectedEnv, expectedEnv2},
+	}
+	cluster := NewTestDockerCluster(t, opts)
+	defer cluster.Cleanup()
+
+	envs := cluster.GetActiveClusterNode().Container.Config.Env
+
+	if !findEnv(envs, expectedEnv) {
```

Comment:
In this test, won't value2 replace value1 for TEST_ENV? How can we obtain different values for the same ENV variable?